chunkymonkey.js

```diff
@@ -15,9 +15,18 @@
 //chunkArrayInGroups([0, 1, 2, 3, 4, 5, 6, 7, 8], 4) should return [[0, 1, 2, 3], [4, 5, 6, 7], [8]].
 
 
+//split entire array into groups the length of size
+//push arrays into an array
 function chunkArrayInGroups(arr, size) {
-  // Break it up.
-  return arr;
+ var array = [];
+  for (var i = 0; i < arr.length; i+=size) {
+    //split an array into groups of given size
+    var temparray = arr.slice(i,i+size);
+  	array.push(temparray);
+  }
+    console.log(array);
```

Comment:
similar to others, it is best to leave the console.log out once you are done with the solution